### PR TITLE
Update hooks-overview.md

### PR DESCRIPTION
**Pull request by amodsachintha**
Changed 'is' -> 'are'. A simple grammar fix.

### CODE REVIEW

File: content/docs/hooks-overview.md [hunk 1]
```diff
@@ -229,7 +229,7 @@ function FriendListItem(props) {
 }
 ```
 
-The state of these components is completely independent. Hooks are a way to reuse *stateful logic*, not state itself. In fact, each *call* to a Hook has a completely isolated state -- so you can even use the same custom Hook twice in one component.
+The state of these components are completely independent. Hooks are a way to reuse *stateful logic*, not state itself. In fact, each *call* to a Hook has a completely isolated state -- so you can even use the same custom Hook twice in one component.
```
Comment (rickhanlonii) [inline content/docs/hooks-overview.md:232]: The noun here is "state" which is singular so we should use "is", but I agree that this is awkward.

What do you think about this:

```suggestion
The state of each component is completely independent. Hooks are a way to reuse *stateful logic*, not state itself. In fact, each *call* to a Hook has a completely isolated state -- so you can even use the same custom Hook twice in one component.
```

Comment (amodsachintha) [reply content/docs/hooks-overview.md:232]: I agree, the latter is much more proper.

**Comment by rickhanlonii** [comment]
Thanks @amodsachintha!

**Comment by amodsachintha** [comment]
> Thanks @amodsachintha!

Thank you too @rickhanlonii !

**Pull request merged**